modules/apps/notification/notification-service/src/main/java/com/liferay/notification/model/impl/NotificationQueueEntryImpl.java

```diff
@@ -19,4 +20,13 @@ public NotificationRecipient getNotificationRecipient() {
 			getNotificationRecipientByClassPK(getNotificationQueueEntryId());
 	}
 
+	@Override
+	public String getClassName() {
+		if (getClassNameId() <= 0) {
+			return "";
+		}
+
+		return PortalUtil.fetchClassName(getClassNameId());
+	}
```

Comment:
@fabioafreitas 

- I couldn't find any other example of this method being overridden in the `mode/impl` package. 
- Since the method is `getClassName`, we should use `PortalUtil.getClassName(getClassNameId())` instead of the `fetch`. 

I believe these two points will lead Brian to reject this change.

Could we try to use a `try/catch` block where the exception is thrown?